File: bench/Reference/file_write_read.erl
#!/usr/bin/env escript
%% Read → write → read → print length.

main(_) ->
    {ok, Data} = file:read_file("bench/data/bench_text.txt"),
    ok = file:write_file("/tmp/erl_bench_write.txt", Data),
    {ok, Data2} = file:read_file("/tmp/erl_bench_write.txt"),
    io:format("~p~n", [byte_size(Data2)]).

File: bench/Reference/file_write_read.py
with open("bench/data/bench_text.txt") as f:
    data = f.read()
with open("/tmp/yona_bench_write_py.txt", "w") as f:
    f.write(data)
with open("/tmp/yona_bench_write_py.txt") as f:
    data2 = f.read()
print(len(data2))
